BE/app/api/routes/script_gen.py

```
@@ -0,0 +1,367 @@
+                message="Celery Task Failure",
+                error=str(task_result.result)
+            )
+            
+        return response
+        
+    except Exception as e:
+        logger.error(f"Status check failed: {e}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"상태 조회 실패: {str(e)}"
+        )
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)